src/components/TopErrorBoundary.tsx

```
@@ -76,9 +79,7 @@ export class TopErrorBoundary extends React.Component<
       if (typeof templateStr === "string") {
         body = encodeURIComponent(templateStr);
       }
-    } catch (error) {
-      console.error(error);
-    }
+    } catch {}
```

why hiding the error?